Tests/CartonCommandTests/InitCommandTests.swift

```diff
@@ -0,0 +1,116 @@
+// Copyright 2020 Carton contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Cavelle Benjamin on Dec/20/20.
+//
+
+@testable import CartonCLI
+import TSCBasic
+import XCTest
+
+extension InitCommandTests: Testable {}
+
+final class InitCommandTests: XCTestCase {
+  func testDefaultArgumentParsing() throws {
+    // given
+    let arguments: [String] = []
+
+    // when
+
+    AssertParse(Dev.self, arguments) { command in
+      // then
+      XCTAssertNotNil(command)
+    }
+  }
+
+  func testHelpString() throws {
```

Comment:
Do we need to assert help command output? I think this test case is fragile and hard to maintain. 
What do you want to ensure by this test case? I think what we want to ensure is that help command outputs "something". We are not interested in the detail contents and it's `ArgumentParser`'s responsibility 😸

Reply:
I would say that we are interested in the details. If we change anything on the command, this should be the first to break and the first to fix. People use the help function and we should ensure it's outputting what we expect.

Reply:
The help output is well tested in [swift-argument-parser package](https://github.com/apple/swift-argument-parser/blob/main/Tests/ArgumentParserUnitTests/HelpGenerationTests.swift), so I think we don't need to re-test it in our test case.

Reply:
I actually identified some other issues with tracking the output text given different system versions. I've removed that test assertion.